packages/edit-site/src/components/template-switcher/style.scss

```diff
@@ -46,3 +47,14 @@
 		display: block;
 	}
 }
+
+.edit-site-template-switcher__theme-preview-name {
+	font-weight: 500;
+	font-size: 140%;
```

Comment:
Any reason for using % value here?  Would something like `$big-font-size` (18px) do the trick?

Reply:
> Any reason for using % value here?

Me being notoriously bad with (S)CSS :grimacing: 

> Would something like `$big-font-size` (18px) do the trick?

I'll give it a try! :+1:

Reply:
14c131d3bc